Krati Agrawal/If-Else/bonus.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+int main()
+{
+    int salary, years;
+    cout << "Enter Salary : ";
+    cin >> salary;
+    cout << "Enter Years of Service : ";
+    cin >> years;
+    if (years > 5)
+    {
+        int bonus = (salary * 5) / 100;
```

Comment:
it should be in float like, salary*0.05, salary*5/100.0, (float)salary*5/100 anything will work. This will give wrong answer for example let say salary is 50, then bonus = 50*5/100 = 250/100 will give you 2 but it should 2.5, and it will be possible in the case of float only.

Reply:
Yes , I've corrected it. plz check!